extensions/ql-vscode/CHANGELOG.md

```diff
@@ -1,11 +1,12 @@
 # CodeQL for Visual Studio Code: Changelog
 
-## 1.1.1
+## 1.1.1 - 23 March 2020
 
+- Fix quick-query not working on `.qll` files.
```

Comment:
Is this from https://github.com/github/vscode-codeql/pull/294? If so, should it be "quick evaluation" rather than "quick query"?

Reply:
Oof, good catch.